musit-service/src/main/scala/no/uio/musit/ws/ProxiedRequest.scala

```diff
@@ -0,0 +1,35 @@
+package no.uio.musit.ws
+
+import no.uio.musit.ws.ProxiedRequest._
+import play.api.Configuration
+import play.api.libs.ws.{DefaultWSProxyServer, WSProxyServer, WSRequest}
+
+class ProxiedRequest(req: WSRequest, config: Configuration) {
+
+  val proxy: Option[WSProxyServer] = for {
+    host <- config.getString(ProxyHost)
+    port <- config.getInt(ProxyPort)
+    user     = config.getString(ProxyUser)
+    password = config.getString(ProxyPassword)
+  } yield
+    DefaultWSProxyServer(host = host, port = port, principal = user, password = password)
+
+  implicit def viaProxy(): WSRequest = proxy.map(req.withProxyServer).getOrElse(req)
```

Comment:
Empty parens are typically used when you have side-effecting function calls like `doStuff()` etc. So if they aren't absolutely necessary the parens should be removed.

Also , I don't think this function needs to be implicit(?).